tests/testsuite/cargo_add/rust_version_older/stderr.log

```diff
@@ -0,0 +1,3 @@
+    Updating `dummy-registry` index
+warning: selecting older version of `rust-version-user` to satisfy this package's rust-version (use `--ignore-rust-version` to override)
```

Comment:
We should note
- `rust-version-user`'s rust-version
- what the latest package's version and rust-version are so people can see how far behind they are and what they would need to upgrade to to get it